### PR TITLE
fix new annotations from previous closed pull request

### PR DESCRIPTION
**Pull request by SvenSchiffner**
* didn't include BLADE anymore, docu for BLADE was added and READING_TIME was changed to INTERVALL_US for asynchronous plugin

### CODE REVIEW

File: x86_energy_plugin.cpp [hunk 1]
```diff
@@ -307,24 +307,14 @@ class x86_energy_plugin
...
-        }
-        /* otherwise use current environment variable */
-        else
-        {
-            env_var = "READING_TIME";
-            def_value = 0;
-        }
-        auto reading_time = static_cast<std::chrono::milliseconds>(
-            stoi(scorep::environment_variable::get(env_var, std::to_string(def_value))));
+        /* get intervall_us and cast ist in the right way */
```
Comment (bmario) [inline x86_energy_plugin.cpp]: > cast it in the right way

File: README.md [hunk 2]
```diff
@@ -242,15 +252,16 @@ E.g.:
...
-    The time in millisecs, between two consecutive reads of the power/energy values. A longer interval
+    The time in microsecs, between two consecutive reads of the power/energy values. A longer interval
     means less disturbance, a shorter interval is more exact.
 
     On Intel CPUs, the registers are updated roughly every msec. If you choose an interval of 1ms
-    you might find highly variating power consumptions. To gain most exact values, you should set
-    the amplitude to 1, if you can live with less accuracy, you should set it to 100.
+    you might find highly variating power consumptions. To gain most exact values
+    relating to temporal granularity, you should set the intervall to 10, but this
+    gives less accure values. If you can live with less granularity, you should
```
Comment (bmario) [inline README.md]: accurate

File: x86_energy_plugin.cpp [hunk 3]
```diff
@@ -140,8 +140,8 @@ class x86_energy_measurement
 {
 public:
     x86_energy_measurement()
-    /* setting default value for reading_time */
-    : mreading_time(std::chrono::milliseconds(5)), is_thread(0)
+    /* setting default value for intervall_us */
+    : mintervall_us(std::chrono::microseconds(50000)), is_thread(0)
```
Comment (bmario) [inline x86_energy_plugin.cpp:144]: Why the change from milliseconds to microseconds? Does that even make sense for RAPL @rschoene
IIRC RAPL has a sampling rate of 1000 Samples/seconds, but they aren't equidistant. This roughly translates to a temporal resolution of 1 millisecond, at most.

**Review by bmario — CHANGES_REQUESTED** [review]
And `interval` is always spelled as `intervall`

File: x86_energy_plugin.cpp (hunk 3 above)
Comment (SvenSchiffner) [reply x86_energy_plugin.cpp:144]: Because the original name, that's used again, is interval_us and not intervall_ms.

Comment (AndreasGocht) [reply x86_energy_plugin.cpp:144]: specially if they are nor equidistant, us make sense, as you can than sample the change of the value.
